pages/how_tos/bitbucket.md.erb

```diff
@@ -30,8 +20,20 @@
 	* Merged
 	* Declined 
```

Comment:
I can see people skimming and skipping the repository event.

Maybe we could break this into two sub-lists, like:

1. From the list of triggers, select:  
   - Repository:  
     * Push
   - Pull Request:  
     * Created
     * Updated
     * Approved
     * Approval Removed
     * Merged
     * Declined